packages/core/src/codewhispererChat/controllers/chat/controller.ts

```diff
@@ -1601,7 +1602,7 @@ export class ChatController {
         //  Do not include chatHistory for requests going to Mynah
         request.conversationState.history = request.conversationState.currentMessage?.userInputMessage?.userIntent
             ? []
-            : this.chatHistoryDb.getMessages(tabID).map((chat) => messageToChatMessage(chat))
+            : this.chatHistoryDb.getMessages(tabID, maxHistoryMessages).map((chat) => messageToChatMessage(chat))
```

Comment:
This might cause invalid history. Also we added a limit of 250 here:
https://github.com/aws/aws-toolkit-vscode/blob/feature/agentic-chat/packages/core/src/shared/db/chatDb/chatDb.ts#L28

Where did this 100 limit come from?